### PR TITLE
Convert search to microservice

### PR DESCRIPTION
**Pull request by Felixim0**

### CODE REVIEW

File: app/downloadshandler.py [hunk 1]
```diff
@@ -20,9 +20,7 @@
     get_fields,
 )
 from app.searchfunctions import (
-    get_all_assignment_status,
-    get_microservice_records,
-)
+    get_microservice_records, )
```
Comment (SammyPoot) [inline app/downloadshandler.py]: If you remove the brackets here, it should all fit on one line cleanly.

File: app/microserviceshandler.py [hunk 2]
```diff
@@ -211,11 +215,17 @@ async def get(self, request):
 
       views, current_view_index = get_views(user_role, microservice_name)
       header_html = get_html(user_role, views)
+      current_view = views[current_view_index]
+      if (current_view.database_name == "search") and (page_number == 1):
+        current_view.hide_table = True
+      elif current_view.database_name == "search" and page_number > 1:
+        current_view.hide_table = False
+        current_view.hide_search_criteria = True
```
Comment (SammyPoot) [inline app/microserviceshandler.py]: The above could be simplified.
```python
current_view = views[current_view_index]
if current_view.database_name == 'search':
  current_view.hide_table = page_number == 1
  current_view.hide_search_criteria = page_number > 1
```

**Review by SammyPoot — CHANGES_REQUESTED** [review]

File: app/downloadshandler.py (hunk 1 above)
Comment (Felixim0) [reply app/downloadshandler.py]: removed

File: app/microserviceshandler.py (hunk 2 above)
Comment (Felixim0) [reply app/microserviceshandler.py]: Have swapped out the logic for this better statement 👌

**Review by SammyPoot — APPROVED** [review]

**Pull request merged**